consensus/mint-client/src/config.rs

```diff
@@ -230,6 +230,27 @@ impl MintTxParams {
 }
 #[derive(Subcommand)]
 pub enum Commands {
+    /// Verify whether a signature over given contents verifies with the
+    /// provided pubkey.
+    #[clap(arg_required_else_help = true)]
+    CheckSig {
+        /// The signature to verify.
+        #[clap(
+            long = "signature",
+            use_value_delimiter = true,
+            parse(try_from_str = load_or_parse_ed25519_signature), env = "MC_MINTING_SIGNATURE"
+        )]
+        signature: Ed25519Signature,
+
+        /// The contents that were signed.
+        #[clap(long, parse(try_from_str = FromHex::from_hex), env = "MC_MINTING_CONTENTS")]
+        contents: Option<[u8; 32]>,
```

Comment:
If we're unwrapping this value immediately in the Commands::CheckSig call we should have the type be non-optional to help with downstream effects and to prevent panics

Reply:
Good catch, this isn't actually optional here

Reply:
Fixed! 🙏